components/HeaderSearch.vue

```diff
@@ -1,29 +1,57 @@
 <template>
-  <div class="headerSearch">
-    <div class="headerSearch__box">
+  <div class="headerSearch" :class="{ headerSearch_homepage: homepage }">
+    <form class="headerSearch__box">
       <div class="headerSearch__inputBox">
         <input
+          v-model="searchText"
           class="headerSearch__input"
           type="search"
           :placeholder="$t('Transaction or Address')"
-        >
+        />
       </div>
       <button class="headerSearch__button" type="button">
         <svg-icon name="common/search" class="headerSearch__icon" />
       </button>
-    </div>
+    </form>
   </div>
 </template>
 
 <script>
 export default {
   name: "HeaderSearch",
+  data: () => ({
+    searchText: '',
+  }),
+  computed: {
+    homepage() {
+      if (this.$route.path === '/') return true
+      if (this.$route.path === '/ru') return true
+      if (this.$route.path === '/en') return true
+      if (this.$route.path === '/de') return true
+      if (this.$route.path === '/tr') return true
```

Comment:
тут явно чет условий многовато

в одну строчку функцию можно написать

еще я бы посмотрел на this.$route.name - там должно быть название для страница вне зависимости от урла
```suggestion
      return /^\/(ru|en|de|tr)?$/.test(this.$route.path)
```